solution/backend/regcore/serializers.py

```diff
@@ -61,3 +62,18 @@ class ParserResultSerializer(serializers.ModelSerializer):
     class Meta:
         model = ECFRParserResult
         fields = '__all__'
+
+
+class SynonymSerializer(serializers.Serializer):
+    baseWord = serializers.CharField()
+    isActive = serializers.BooleanField()
+
+
+class SynonymsSerializer(serializers.Serializer):
+    isActive = serializers.BooleanField()
+    baseWord = serializers.CharField()
+    synonyms = SynonymSerializer(read_only=True, many=True)
+
+    class Meta:
+        model = Synonym
+        fields = ('__all__')
```

Comment:
Since this is a `serializers.Serializer` and not a `ModelSerializer`, you may not need the Meta class.